openrtb_ext/bidders_validate_test.go

```diff
@@ -24,6 +24,9 @@ var validator BidderParamValidator
 // returns valid JSON for all known CoreBidderNames.
 func TestBidderParamSchemas(t *testing.T) {
 	for _, bidderName := range CoreBidderNames() {
+		if _, ok := GetAliasBidderNamesSet()[bidderName]; ok {
+			continue
+		}
```

Comment:
```go
if HasAliasBidderNames(bidderName) {
	continue
}
```

Reply:
Don't think we need to create a separate function as `HasAliasBidderNames` would be just one liner.